README.md

```diff
@@ -18,8 +20,10 @@ These templates are not an exhaustive list of use cases or possibilities that de
 There are two ways you can use the templates provided in this repository. 
 
 1. Using SnowCLI (recommended)
+
+    Please refer to the online Snowflake Documentation on installing and using SnowCLI to create Native Applications for more information. 
```

Comment:
nit: Maybe it's just me, but I had to read this a few times for my english to parser to get it. Something like this might be clearer IMHO:
For more information, please refer to [...]